shell/components/Inactivity.vue

```diff
@@ -72,9 +73,18 @@ export default {
       this.clearAllTimeouts();
       const endTime = Date.now() + this.showModalAfter * 1000;
 
+      this.id = endTime;
+      this.$store.dispatch('performance/setId', this.id);
```

Comment:
A whole store is pretty heavy handed for this. `let globalId` above the `export` should work the same without the overhead.